components/core/tools/scripts/lib_install/centos-stream-9/install-prebuilt-packages.sh

```
@@ -23,29 +19,9 @@ dnf install -y \
-# NOTE: We lock `task` to a version < 3.43 to avoid https://github.com/y-scope/clp/issues/872
-task_pkg_path="$(mktemp -t --suffix ".rpm" task-pkg.XXXXXXXXXX)"
-curl \
-    --fail \
-    --location \
-    --output "$task_pkg_path" \
-    --show-error \
-    "https://github.com/go-task/task/releases/download/v3.42.1/task_linux_${task_pkg_arch}.rpm"
-dnf install --assumeyes "$task_pkg_path"
-rm "$task_pkg_path"
+# Install `cmake`, `go-task` and `uv`
+"${script_dir}/../pipx_install/install-all.sh"
```

Shellcheck (0.10.0)

[warning] script_dir is referenced but not assigned.

(SC2154)